docs/tutorial.rst

```diff
@@ -1256,7 +1256,7 @@ as aliases, by wrapping them in ``@()``.  For example:
 
     >>> @(_banana)
     'My spoon is tooo big!'
-    >>> echo "hello" | @(lambda args, stdin=None: stdin.strip() + args[0]) world
```

Comment:
Even if previous Xonsh version might have passed something else than an `_io.TextIOWrapper` as `stdin` or an `_io.TextIOWrapper` might have behaved differently (more string-like), I don't quite see how this implementation ever resultet in the desired output with a space between the words and an `EOL` at the end, so the prompt wouldn't eat the output on the next CLI input ... :confused: